x-pack/plugins/observability_solution/synthetics/server/routes/settings/dynamic_settings.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+import { schema } from '@kbn/config-schema';
+import { savedObjectsAdapter } from '../../saved_objects';
+import { SyntheticsRestApiRouteFactory } from '../types';
+import { DynamicSettings } from '../../../common/runtime_types';
+import { DynamicSettingsAttributes } from '../../runtime_types/settings';
+import { SYNTHETICS_API_URLS } from '../../../common/constants';
+
+export const createGetDynamicSettingsRoute: SyntheticsRestApiRouteFactory<
```

Comment:
Question: Should any of these new routes be protected with access tags?

Reply:
@kc13greiner app has logic to by default add write tags to all non-GET routes